lib/appender/standard.js

```diff
@@ -111,8 +111,13 @@ function standardLogAppender(level, exception, message, data) {
   // If stack is not defined we are in a browser that does not fully support our logging.
   // Or one of our custom loggers was called, e.g. Logger.debug('message').
   if (exception === undefined || exception === null || exception.stack === null) {
-    var scripts = window.document.getElementsByTagName('script');
-    scriptURL = (window.document.currentScript || scripts[scripts.length - 1]).src;
+    if(typeof window.document.getElementsByTagName === 'function') {
```

Comment:
Since `window` is undefined in a node context (unless someone defines it), would it be simpler to check if `window` is undefined?  

Per http://stackoverflow.com/questions/4224606/how-to-check-whether-a-script-is-running-under-node-js, it also looks like you can check for common support.

@garrypolley, do you have a preference?

-------

This is just a nitpick, but I'd like a space between the `if` and the opening paren.